service/service_test.go

```diff
@@ -137,7 +137,7 @@ func TestApplication_StartAsGoRoutine(t *testing.T) {
 	require.NoError(t, err)
 
 	params := Parameters{
-		ApplicationStartInfo: ApplicationStartInfo{
+		ApplicationStartInfo: component.ApplicationStartInfo{
```

Comment:
Shall we extract this into a function in test package, e.g. `componenttest`, and use it everywhere instead of creating empty struct?

```go
package componenttest

func TestApplicationStartInfo() component.ApplicationStartInfo {
  return component.ApplicationStartInfo{
    // Default test values
    ...
  }
}
```